platform/ddf-branding-plugin/pom.xml

```diff
@@ -17,17 +17,16 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <artifactId>distribution</artifactId>
-        <groupId>ddf</groupId>
+        <artifactId>platform</artifactId>
+        <groupId>ddf.platform</groupId>
         <version>2.11.0-SNAPSHOT</version>
     </parent>
-    <groupId>ddf.distribution</groupId>
     <artifactId>ddf-branding-plugin</artifactId>
```

Comment:
This `artifactId` could be `platform-branding-plugin` instead to match the other modules in ddf/platform.